docs/source/examples/pt_zne.md

```diff
@@ -0,0 +1,406 @@
+```
+
+```{caution}
+It is worth noting that Pauli twirling's goal is to tailor the noise from coherent to incoherent. 
+
+Depending on the noise strength, type of coherent noise etc. this transformation might not give better results after the Pauli twirled circuit is executed. See the plot in the next section for an example.
+```
+
+## Combining Pauli Twirling with ZNE
+
+```{code-cell} ipython3
+from mitiq.zne import execute_with_zne
+
+executor=partial(execute, noise_level=NOISE_LEVEL)
+zne_pt_vals = []
+
+for twirled_circuit in twirled_circuits:
+    zne_pt_vals.append(execute_with_zne(twirled_circuit, executor))
+
+mitigated_result = np.average(zne_pt_vals)
+
+print(f"Error without twirling: {abs(ideal_value - noisy_value) :.3}")
+print(f"Error with twirling: {abs(ideal_value - twirled_result) :.3}")
+print(f"Error with ZNE + PT: {abs(ideal_value - mitigated_result) :.3}")
+
+```
+Accordingly, depending on the noise strength, a combination of PT and ZNE do not work that well compared to just PT or ZNE. Thus, it is important to understand when combining a noisy tailoring technique with an error mitigation technique provides a significant advantage. 
+
+```{code-cell} ipython3
+
+import matplotlib.pyplot as plt
+
+# Plot error vs noise strength
+noise_strength = np.linspace(0.0, 1.0, 50)
+error_no_zne_no_twirling = []
+error_with_twirling = []
+error_with_twirling_and_zne = []
+zne_vals = []
+ideal_values = []
+NUM_TWIRLED_VARIANTS = 30
+
+for strength in noise_strength:
+    ideal_value = execute(circuit, noise_level=0.0)
+    ideal_values.append(ideal_value)
+    noisy_value = execute(circuit, noise_level=strength)
+    id_noisy_diff = abs(ideal_value-noisy_value)
+    error_no_zne_no_twirling.append(id_noisy_diff)
+
+    twirled_circuits = generate_pauli_twirl_variants(circuit, num_circuits=NUM_TWIRLED_VARIANTS)
+    pt_vals = Executor(partial(execute, noise_level=strength)).evaluate(twirled_circuits)
+    twirled_result = np.average(pt_vals)
+    twirled_noisy_diff = abs(ideal_value-twirled_result)
+    error_with_twirling.append(twirled_noisy_diff)
+
+    executor=partial(execute, noise_level=strength)
+    zne_vals.append(abs(ideal_value - execute_with_zne(circuit, executor)))
+    
+    zne_pt_vals = []
+    for i in twirled_circuits:
+        zne_pt_vals.append(execute_with_zne(i, executor))
+    mitigated_twirled_result = np.average(zne_pt_vals)
+    error_with_twirling_and_zne.append(abs(ideal_value - mitigated_twirled_result))
+
+
+plt.plot(noise_strength, error_no_zne_no_twirling,"", label=r"|Ideal - Noisy|", color="#1f77b4")
+plt.plot(noise_strength, zne_vals, "", label=r"|Ideal - ZNE|", color="#bcbd22")
+plt.plot(noise_strength, error_with_twirling,"", label=r"|Ideal - Twirling|", color="#ff7f0e")
+plt.plot(noise_strength, error_with_twirling_and_zne, "", label=r"|Ideal - (ZNE + Twirling)|", color="#2ca02c")
+
+plt.xlabel(r"Noise strength, Coherent noise:$R_y(\frac{\pi}{2} \times \text{noise_strength})$")
```

Comment:
nit: «Noise strength» as label vs. noise_strength as variable. Use the same, so it's clear that it's the actual values in the axis that parametrize the noise operation.